src/main/java/org/apache/commons/lang3/builder/ReflectionToStringBuilder.java

```diff
@@ -867,4 +874,21 @@ private void validate() {
         }
     }
 
+    private boolean handleNativeClasses() {
+        Object value = getObject();
+        if (value instanceof Number || value instanceof Boolean || value instanceof Character
+            || value instanceof TemporalAccessor ) {
+            getStringBuffer().append("value=").append(getObject().toString());
+            return true;
+        }
+        
```

Comment:
Extra blank line not needed IMO. Want I want call attention to something, I write a // comment.